packages/browser-pool/tab-as-a-container/background.js

```diff
@@ -204,14 +204,13 @@ const getNextLocalhostIp = (openerId) => {
 
     localhostIpCache.set(openerId, result);
 
-    // 255 * 255 * 255 = 16 581 375
-    if (localHostIp[3] === 255) {
-        if (localHostIp[2] === 255) {
-            if (localHostIp[1] === 255) {
+    if (localHostIp[3] === 254) {
+        if (localHostIp[2] === 254) {
+            if (localHostIp[1] === 254) {
                 localHostIp[0] = 127;
```

Comment:
Is this first element ever going to change? I assume not

Reply:
No but again this is for visibility. It's easier to read `127.0.0.1` than `0.0.1`

Reply:
Actually there was a bug, I forgot to set to 0 other parts. PTAL